org.eclipse.scout.rt.jackson/src/main/java/org/eclipse/scout/rt/jackson/dataobject/ScoutDataObjectModule.java

```diff
@@ -125,9 +141,4 @@ public void setupModule(SetupContext context) {
   public int hashCode() {
     return NAME.hashCode();
   }
-
```

Comment:
Should hashCode be removed as well then? Not sure when this is used.

Reply:
Seems unused as well, done.